astropy/modeling/mappings.py

```diff
@@ -59,8 +60,8 @@ def __init__(self, mapping, n_inputs=None, name=None, meta=None):
         self._n_outputs = len(mapping)
         super().__init__(name=name, meta=meta)
 
-        self.inputs = tuple('x' + str(idx) for idx in range(self._n_inputs))
-        self.outputs = tuple('x' + str(idx) for idx in range(self._n_outputs))
+        self.inputs = tuple("x" + str(idx) for idx in range(self._n_inputs))
+        self.outputs = tuple("x" + str(idx) for idx in range(self._n_outputs))
```

Comment:
Many lines have been changed only because `black` has changed the single-quoted strings to double-quoted strings. PEP 8 says either is fine as long as the choice is adhered to (https://peps.python.org/pep-0008/#string-quotes). Is there a way to tell `black` to prefer single-quotes over double-quotes and would that result in fewer lines being modified?

Reply:
Or maybe just leave them alone? We don't have to apply black to everything, do we?

Reply:
We can sort-of do that with [`--skip-string-nomalization`](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html?highlight=quotes#string); however, rather than preferring one string quote style over the other, it instead does not modify string quotes at all. 

My personal opinion is that we should strive for as much consistency in the codebase as possible if we go the route of using black; however, I will defer to the community's consensus.

Reply:
> Or maybe just leave them alone? We don't have to apply black to everything, do we?

Technically, if someone really does not like what black is doing they can wrap the section of code in:
```python
# fmt: off
...
# fmt: on
```
or add `# fmt: skip` to a single line. This is also a very-much opt-in change, even at the single file level (the configuration does allow one to exclude any single file or folder from black formatting).

Reply:
While it might be useful as an initial step to look at the smaller diff set with `--skip-string-normalization`, at the end I would argue to fully commit to default black settings. This is the point of never arguing about style again.

When you are writing code you can still type single quotes (which is a little faster) and the pre-commit hooks will fix them fairly painlessly.

Reply:
Note that PEP8 says "Pick a rule and stick to it." But astropy core does not pick a rule and stick to it. The choice is basically random.